internal/build/make_coverage.sh

```
@@ -0,0 +1,67 @@
+    echo "Error: Index file not found: $INDEX" >&2
+    echo "Run 'make .tmp/index' first" >&2
+    exit 1
+fi
+
+# Create coverage directory
+mkdir -p "$COVERAGE_DIR"
+
+# Run tests for each module using make targets
+echo "Running coverage tests..."
+n=1
+while IFS=: read -r name dir mod deps; do
```

Shellcheck (0.10.0)

[warning] dir appears unused. Verify use (or export if used externally).

(SC2034)